web/docs/glossary.md

```diff
@@ -0,0 +1,170 @@
+offers a transactional interface for adding and removing partitions.
+
+- [Tune catalog
+  fragmentation](./setup-guides/tune-performance/README.md#tune-catalog-fragmentation)
+- [Configure catalog and partition indexes](./setup-guides/tune-performance/README.md#configure-catalog-and-partition-indexes)
+
+## Connector
+
+Manages chunks of raw bytes by interacting with a resource.
+
+A connector is either a *loader* that acquires bytes from a resource, or a
+*saver* that sends bytes to a resource. Example connectors are
+[`file`](./connectors/file.md), [`kafka`](./connectors/kafka.md), and
+[`nic`](./connectors/nic.md).
+
+- See all available [connectors](./connectors.md)
+
+## Format
+
+Translates between bytes and events.
+
+A format is either a *parser* that converts bytes to events, or a *printer*
+that converts events to bytes. Example formats are [`json`](./formats/json.md),
+[`cef`](./formats/cef), and [`pcap`](./formats/pcap.md).
+
+- See all available [formats](./formats.md)
+
+## Index
+
+Optional data structures for accelerating historical queries.
+
+Tenzir has *sparse* and *dense* indexes. Sparse indexes live in memory and point
+to [partitions](#partition), whereas dense indexes live within a partition and
+point to individual rows within the partition [store](#store).
+
+- [Configure catalog and partition indexes](./setup-guides/tune-performance/README.md#configure-catalog-and-partition-indexes)
+
+## Node
+
+A host for [pipelines](#pipeline) and storage reachable over the network.
+
+The `tenzir-node` binary starts a node in a dedicated server process that
+listens on TCP port 5158.
+
+- [Deploy a node](./setup-guides/deploy-a-node/README.md)
+- [Use the app](./setup-guides/use-the-app/README.md) to manage a node
+- Use the [REST API](./rest-api.md) to manage a node.
+- [Import into a node](./user-guides/import-into-a-node.md)
+- [Export from a node](./user-guides/export-from-a-node.md)
+
+## Metrics
+
+Runtime statistics about pipeline and node execution.
+
+- [Collect metrics](./setup-guides/collect-metrics.md)
+- See all available [metrics](./metrics.md) details
+
+## Operator
+
+The atomic building block of a [pipeline](#pipeline).
+
+An operator is a [source](#source), [transformation](#transformation), or
+[sink](#sink)
+
+## Partition
+
+The horizontal scaling unit of the storage attached to a [node](#node).
+
+A partition contains a [store](#store) and optionally a set of indexes.
+
+## Pipeline
+
+Combines a set of [operators](#operator) into a dataflow graph.
+
+- Understand the [pipeline language](./language/pipelines.md)
+- [Run a pipeline](./user-guides/run-a-pipeline/README.md)
+
+## Platform
+
+Control plane for nodes and pipelines, accessible through [app](#app) at
+[app.tenzir.com](https://app.tenzir.com).
+
+- [Use the app](./setup-guides/use-the-app/README.md)
+
+## Schema
+
+A named record type describing the top-level structure of a data frame.
```

Comment:
We do not use the word data frame in our code base or in our documentation. It's a bit of an odd encounter here. Can we describe this differently?

Reply:
We actually use it in the first sentence here: https://docs.tenzir.com/next/data-model/schemas. I kept the definition synchronous. Users probably understand it, although "data frame" is not (yet) a first-class abstraction.